example/example.documentation.yaml

```diff
@@ -267,16 +255,13 @@ paths:
                 type: object
                 properties:
                   status:
-                    type: string
                     const: created
                   data:
                     type: object
                     properties:
                       id:
                         type: integer
-                        format: int64
-                        exclusiveMinimum: 0
-                        maximum: 9007199254740991
+                        exclusiveMaximum: 9007199254740991
```

Comment:
this is still wrong in Zod (external bug)

Reply:
missing `0` would be fixed by this PR:
https://github.com/colinhacks/zod/pull/4267

Reply:
the issue regarding maximum: https://github.com/colinhacks/zod/issues/4309

Reply:
fixed